### ISSUE TITLE
Negative value broke everything

### ISSUE THREAD

**Issue by Gabbosenpai** [digitalshadowhawk/pf2e-subsystem-helper#7]
v0.0.11

After putting a negative value on an influence threshold on a Influence stat block, the subsystem page became all blank, making the whole module not working at all.

1. Go to 'Actors->Party'
2. Click on 'Open Sheet'
3. Go to 'Subsystems'
4. Create a New NPC for the Influence Subsystem
5. Add new threshold
6. Set its value to -1
7. Everything disappears

**Expected behavior**
I expected to see either -1 on the threshold value or an error saying that you cannot use this value or something else

**Screenshots**
![immagine_2024-12-12_152606835](https://github.com/user-attachments/assets/3ea1c326-3d9c-4670-b783-f7bb220bf4c6)

**Browser:**
 - Foundry Application, not tried on browser

**Foundry Version: 12 Stable 331** 

Pathfinder Second Edition - Version 6.7.2

**Comment by ghostphoenix713** [comment]
I'm having a similar issue currently. I didn't try to add something with a negative value, but I just lost everything in the Subsystem section of the party sheet with no warning. I've tried turning off and on specific subsystems and turning off the module and turning it back on altogether, but neither did anything to bring the already input information back. My party sheet looks exactly like the screenshot posted by @Gabbosenpai now. I'd put a good deal of work into the Influence section and now it's just all gone and that is incredibly frustrating.

**Comment by FastJack-MD** [comment]
I've ran into this as well by entering a zero in the Research Thresholds. Any help would be great!

**Comment by ghostphoenix713** [comment]
I'd just like a way to clear it out and start from scratch at this point. I'm surprised that removing the module and reinstalling does nothing at all

**Comment by FastJack-MD** [comment]
Agreed. I've tried to find where the information is being saved, but I think it's in the cloud because I don't see anything locally.
